package/govuk-prototype-kit.config.json

```diff
@@ -1,14 +0,0 @@
-{
```

Comment:
Not clear to me why this file has been deleted, shouldn't it stay until it is replaced in the next release?

Reply:
@lfdebrux Good shout, it gets overwritten like everything else in `./package` 👍